tests/infra/runner.py

```diff
@@ -44,6 +44,8 @@ def filter_nodes(primary, backups, filter_type):
     if filter_type == "primary":
         return [primary]
     elif filter_type == "backups":
+        if len(backups) == 0:
```

Comment:
```suggestion
        if not backups:
```

https://www.python.org/dev/peps/pep-0008/#programming-recommendations